### DISCUSSION TITLE
How to export animation.ma file to USD

### DISCUSSION THREAD

**Discussion by goutmaster** [Autodesk/maya-usd#2408] [Q&A]
Hi guys,

I am very new at USD format and I am facing exporting USD issue in Maya.

Here is my situation.
- rig and animation is done in Maya
- export it as USD
- bring that USD into Maya Stage or in Solaris Stage as well, but both are fail.
There is geometry, but there is no animation data.

What did I miss?

And I am using Maya 2002.1

**Comment by santosd** [comment]
Hello @goutmaster, what version of Maya and MayaUSD are you using? Is animation enabled in the export options?
